vendor/assets/stylesheets/dvl/core/dropdowns.scss

```diff
@@ -101,6 +101,10 @@ span.dropdown {
     font-weight: normal;
     padding: 0.25rem 0.5rem;
     display: block;
+    font-size: 0.9rem;
+  }
+  .control_styled {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `control_styled` should be written in lowercase with hyphens